service/create/service.go

```diff
@@ -298,10 +313,11 @@ type runMachinesInput struct {
 	prefix              string
 }
 
-func (s *Service) runMachines(input runMachinesInput) error {
+func (s *Service) runMachines(input runMachinesInput) ([]*string, error) {
```

Comment:
This problem in general applies also to my summary of the review. I think the design here is kind of broken when we change the interface to get information after creating resources.